### ISSUE TITLE
The ‘data’ object for properties/process REST API comes back empty 

### ISSUE THREAD

**Issue by exp-cpandya** [sapia-oss/corus#24]
For Corus’ properties/process API the ‘data’ object in 4.7 comes back empty while it was coming back filled-in in 4.5.
 
##### Host 1
```
[
    {
        "cluster": "lpso-ci-ssl-hg",
        "host": "10.208.82.193:33000",
        "data": [ ]
    }
]
```
##### Host 2
```
[
    {
        "cluster": "lpso-ci-ssl-hg",
        "host": "10.208.57.225:33000",
        "data": [ ]
    }
]
```

#### Corus 4.5
```
[
    {
        "cluster": "lpso-qa3-ssl-hg",
        "host": "10.184.25.220:33000",
        "data": 
        [
            {
                "name": "GEMFIRE",
                "value": "/opt/app/gemfire/current"
            },
            {
                "name": "GEMFIRE_SERVER_LOG",
                "value": "/var/log/gemfire/server_log/"
            },
            {
                "name": "corus.env",
                "value": "sslqa3"
            },
            {
                "name": "gemfire.ConcGCThreads",
                "value": "6"
            },
            {
                "name": "gemfire.ParallelGCThreads",
                "value": "64"
            },
            {
                "name": "gemfire.group",
                "value": "Expedia_SSL"
            },
            {
                "name": "gemfire.locators",
                "value": "chelslgfcqa301[43111],chelslgflqa302[43111]"
            },
            {
                "name": "gemfire.lport",
                "value": "43111"
            },
            {
                "name": "gemfire.mem.ccs",
                "value": "512m"
            },
            {
                "name": "gemfire.mem.loc",
                "value": "1g"
            },
            {
                "name": "gemfire.mem.ms",
                "value": "10g"
            },
            {
                "name": "gemfire.mem.mx",
                "value": "10g"
            },
            {
                "name": "gemfire.mem.ng",
                "value": "2g"
            },
            {
                "name": "gemfire.mem.ps",
                "value": "256m"
            },
            {
                "name": "gemfire.mem.ss",
                "value": "384k"
            },
            {
                "name": "max.gemfire-jmx",
                "value": "45004"
            },
            {
                "name": "max.gemfire-sport",
                "value": "60004"
            },
            {
                "name": "min.gemfire-jmx",
                "value": "45001"
            },
            {
                "name": "min.gemfire-sport",
                "value": "60001"
            },
            {
                "name": "service.ConcGCThreads",
                "value": "6"
            },
            {
                "name": "service.ParallelGCThreads",
                "value": "64"
            },
            {
                "name": "service.threads",
                "value": "100"
            },
            {
                "name": "skynet.locator.1",
                "value": "chelslgfcqa307"
            },
            {
                "name": "skynet.locator.2",
                "value": "chelslgfcqa308"
            },
            {
                "name": "skynet.locator.port",
                "value": "43113"
            },
            {
                "name": "tomcat.acceptCount",
                "value": "200"
            },
            {
                "name": "tomcat.connectionTimeout",
                "value": "20000"
            },
            {
                "name": "tomcat.home",
                "value": "/opt/app/tomcat7/current"
            },
            {
                "name": "tomcat.maxHttpHeaderSize",
                "value": "8192"
            },
            {
                "name": "tomcat.maxThreads",
                "value": "750"
            },
            {
                "name": "tomcat.mem.ccs",
                "value": "512M"
            },
            {
                "name": "tomcat.mem.ms",
                "value": "10G"
            },
            {
                "name": "tomcat.mem.mx",
                "value": "10G"
            },
            {
                "name": "tomcat.mem.ng",
                "value": "2G"
            },
            {
                "name": "tomcat.mem.ps",
                "value": "256M"
            },
            {
                "name": "tomcat.mem.ss",
                "value": "384K"
            },
            {
                "name": "tomcat.minSpareThreads",
                "value": "400"
            }
        ]
    }
]
```

**Comment by yduchesne** [comment]
Will reproduce and fix ASAP

**Comment by yduchesne** [comment]
Just realized this had been fixed in a past 4.7 RC release. Closing
